### DISCUSSION TITLE
fatal: repository 'https://github.com/the-fast-track/book-6.2.1/' not found

### DISCUSSION THREAD

**Discussion by ghost** [symfony/symfony#54246] [Symfony Book: The Fast Track] [answered]
Hi, I got the repository not found error when running `symfony new --version=6.2.1 --book guestbook`, there is no 2FA enabled for my Github account and the Personal access tokens have every access permission enabled. Any ideas?

**Comment by MatTheCat** [comment, marked as answer]
Hello, the version is `6.2-1`, not `6.2.1`.

**Comment by ghost** [comment]
Thank you, I feel embarrassed, but again thank you.
